ui/imports/shared/views/ProfileDialogView.qml

```diff
@@ -585,13 +589,17 @@ Pane {
                         placeholderTextColor: Theme.palette.directColor1
                         edit.readOnly: true
                         rightComponent: StatusButton {
+                            id: copyLinkBtn
                             anchors.verticalCenter: parent.verticalCenter
                             borderColor: Theme.palette.primaryColor1
                             size: StatusBaseButton.Size.Tiny
                             text: qsTr("Copy")
                             onClicked: {
                                 text = qsTr("Copied")
                                 root.profileStore.copyToClipboard(d.linkToProfile)
+                                d.timer.setTimeout(function() {
+                                    copyLinkBtn.text = qsTr("Copy")
```

Comment:
Suggestion: You could add a private `d` property and use it in both places (line 596 `text: qsTr(xx)` and here) to avoid posible mismatches

Reply:
I agree, I even think this could be refactored to a shared component. Feel free to create an issue for it :)